genref/go.mod

```diff
@@ -3,31 +3,35 @@ module github.com/kubernetes-sigs/reference-docs/genref
 go 1.16
 
 require (
-	github.com/google/go-cmp v0.5.2 // indirect
+	github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96 // indirect
+	github.com/go-openapi/spec v0.0.0-20160808142527-6aced65f8501 // indirect
+	github.com/google/go-cmp v0.5.5 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/kr/text v0.2.0 // indirect
-	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e
 	github.com/pkg/errors v0.9.1
-	github.com/tengqm/kubeconfig v0.0.0-20210703133455-61479ab64f9d
-	github.com/yuin/goldmark v1.2.1
+	github.com/tengqm/kubeconfig v0.0.0-20210707141017-7849c7b17290 // indirect
```

Comment:
I'm not sure what this new indirect dependency (at version 0.0.0) is doing / why it's included. It stands out as different from the others.

Reply:
I guess this is okay.

Reply:
the `go mod` command sometimes adds the `// indirect` comment, sometimes it doesn't.